src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixAbstractMethodsStep.cs

```diff
@@ -19,43 +19,79 @@ namespace MonoDroid.Tuner
 	/// <summary>
 	/// NOTE: this step is subclassed so it can be called directly from Xamarin.Android.Build.Tasks
 	/// </summary>
-	public class FixAbstractMethodsStep : BaseStep
+	public class FixAbstractMethodsStep
+#if NET5_LINKER
+	: IMarkHandler
+	{
```

Comment:
This structure is likely to "confuse"certain editors that "count" braces w/o knowing about `#if` precedence.

Please keep braces either fully "in" or fully "out", e.g.

```csharp
public class FixAbstractmethodsStep :
#if NET5_LINKER
    IMarkHandler
#else   // !NET5_LINKER
    BaseStep
#endif  // !NET5_LINKER
{

#if NET5_LINKER
    protected LinkContext Context { get; private set; }
    protected AnnotationStore Annotations => Context?.Annotations;
    // ...
#endif  // !NET5_LINKER
}
```